mypyc/genops.py

```diff
@@ -1001,9 +1048,17 @@ def visit_class_def(self, cdef: ClassDef) -> None:
         self.allocate_class(cdef)
 
         for stmt in cdef.defs.body:
-            if isinstance(stmt, (FuncDef, Decorator, OverloadedFuncDef)):
+            if isinstance(stmt, (FuncDef, Decorator)):
                 with self.catch_errors(stmt.line):
                     self.visit_method(cdef, get_func_def(stmt))
+            elif isinstance(stmt, (OverloadedFuncDef)):
```

Comment:
Maybe make the first case be `if isinstance(stmt, (OverloadedFuncDef)) and stmt.is_property`, which lets us avoid duplicating the main case?